### DISCUSSION TITLE
does not matter - to delete

### DISCUSSION THREAD

**Discussion by dudus2000** [dckiller51/bodymiscale#115] [General]
does not matter

**Comment by dckiller51** [comment]
![Russian 1](https://user-images.githubusercontent.com/53062806/184592149-c2161fd0-d02b-4cbd-8f03-e22a96665209.jpg)
![Russian 2](https://user-images.githubusercontent.com/53062806/184592165-a42dba25-be18-4bda-bef2-faa20646ef11.jpg)
